Rakefile

```diff
@@ -22,7 +22,9 @@ end
 
 desc "Given a title as an argument, create a new post file"
 task :new_post, [:title] do |t, args|
-  filename = "#{Time.now.strftime('%Y-%m-%d')}-#{args.title.gsub(/\s/, '_').downcase}.md"
+  title = ENV["title"] || "new-post"
```

Comment:
> Why use `ENV['title']` here instead of `args.title` ?

Yeah I just copy-pasted from my own blog Rakefile, the old code was not
working for me at all. Will change that.

-- 
Later,
 Lukas #lzap Zapletal